### PR TITLE
Added card component and style

### PR DESCRIPTION
**Pull request by Pushpa88431**
sync dev to card_component

### CODE REVIEW

**Comment by Pushpa88431** [comment]
Merged

**Comment by JerryUusis** [comment]
I'll review this today

File: frontend/src/App.css [hunk 1]
```diff
@@ -187,6 +186,71 @@ body {
...
+.grid-item {
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  font-size: 15px;
+  text-align: center;
+}
+.pageHeader {
+  font-size: 35px;
+  color: brown;
```
Comment (JerryUusis) [inline frontend/src/App.css:254]: Brown isn't on the color palette that's been defined at the first row of the CSS file.

File: frontend/src/App.css [hunk 2]
```diff
@@ -187,6 +186,71 @@ body {
...
+  object-fit: cover;
+  width: 100%;
+}
+
+.card h2 {
+  margin: 0;
+  padding: 0.5rem;
+}
+
+.card .content {
```
Comment (JerryUusis) [inline frontend/src/App.css:219]: This class isn't used anywhere

File: frontend/src/App.css [hunk 3]
```diff
@@ -187,6 +186,71 @@ body {
...
+  padding: 0.5rem;
+}
+
+.card footer {
+  background-color: rgb(3, 60, 9);
```
Comment (JerryUusis) [inline frontend/src/App.css:224]: This color isn't from the color palette. Please use CSS variable `--hunter-green`.

File: frontend/src/App.css [hunk 4]
```diff
@@ -187,6 +186,71 @@ body {
...
+  color: #fff;
+  height: 20px;
+  width: 300px;
```
Comment (JerryUusis) [inline frontend/src/App.css:227]: Using relative units for child elements is a better way to make sure they are right size. For example here `width: 100%` will work better.

File: frontend/src/App.css [hunk 5]
```diff
@@ -187,6 +186,71 @@ body {
...
+}
+
+.card img {
+  object-fit: cover;
+  width: 100%;
+}
+
+.card h2 {
+  margin: 0;
```
Comment (JerryUusis) [inline frontend/src/App.css:215]: There isn't a h2 element on the card. `margin: 0` has been declared with the universal selector on line 18.

**Review by JerryUusis — CHANGES_REQUESTED** [review]
The branch feature is supposed to be Card component so I was expeceting a Card component and a few selectors in CSS. Now there's closer to 60 lines of code added to CSS.

There is quite a chunk of unused code here. Let's not push unused code on the dev/main branches. Consider refactoring the changes made in to App.css instead of trying to fix them one by one.

**Pull request closed without merging**